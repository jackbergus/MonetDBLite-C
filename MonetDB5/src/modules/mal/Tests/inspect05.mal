#Illustrate the modules loaded after start
k:= inspect.getKind();
m:= inspect.getModule();
f:= inspect.getFunction();
algebra.sort(f);
s:= inspect.getSignatures();
a:= inspect.getAddresses();
io.table(f,k,m,s,a);
