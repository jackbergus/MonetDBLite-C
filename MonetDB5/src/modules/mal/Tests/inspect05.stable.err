stderr of test 'inspect05` in directory 'src/modules/mal` itself:


# 07:36:14 >  
# 07:36:14 >   mserver5 "--config=/ufs/mk/monet5/Linux/etc/monetdb5.conf" --debug=10 --set "monet_mod_path=/ufs/mk/monet5//Linux/lib/MonetDB5:/ufs/mk/monet5//Linux/lib/MonetDB5/lib:/ufs/mk/monet5//Linux/lib/MonetDB5/bin" --set "gdk_dbfarm=/ufs/mk/monet5//Linux/var/MonetDB5/dbfarm" --set "sql_logdir=/ufs/mk/monet5//Linux/var/MonetDB5/sql_logs"  --set mapi_open=true --set xrpc_open=true --set mapi_port=38805 --set xrpc_port=40401 --set monet_prompt= --trace  --dbname=mTests_src_modules_mal  inspect05.mal
# 07:36:14 >  

#warning: please don't forget to set your vault key!
#(see /ufs/mk/monet5/Linux/etc/monetdb5.conf)

# 07:36:14 >  
# 07:36:14 >  Done.
# 07:36:14 >  

