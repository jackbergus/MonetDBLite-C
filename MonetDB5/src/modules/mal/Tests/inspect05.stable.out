stdout of test 'inspect05` in directory 'src/modules/mal` itself:


# 00:03:55 >  
# 00:03:55 >  Mtimeout -timeout 60 Mserver "--config=/ufs/mk/monet5/src/mal/Tests/All.conf" --debug=10 --set "monet_mod_path=/ufs/mk/monet5/Linux/lib/MonetDB5:/ufs/mk/opensource/MonetDB/Linux/lib/MonetDB" --set "gdk_dbfarm=/ufs/mk/opensource/MonetDB/Linux/var/MonetDB5/dbfarm" --set "sql_logdir=/ufs/mk/opensource/MonetDB/Linux/var/MonetDB5/log" --set mapi_port=33330 --set sql_port=49911 --set xquery_port=56575 --set monet_prompt= --trace --dbname=mTests_src_modules_mal  inspect05.mal </dev/null
# 00:03:55 >  

# Monet Database Server V4.99.19
# Copyright (c) 1993-2005, CWI. All rights reserved.
# Compiled for x86_64-redhat-linux-gnu/64bit with 32bit OIDs; dynamically linked.
# config:/ufs/mk/monet5/src/mal/Tests/All.conf
# dbfarm:/ufs/mk/opensource/MonetDB/Linux/var/MonetDB5/dbfarm
# dbname:mTests_src_modules_mal
# Visit http://monetdb.cwi.nl/ for further information.
##Illustrate the modules loaded after start
#k:= inspect.getKind();
#m:= inspect.getModule();
#f:= inspect.getFunction();
#s:= inspect.getSignature();
#io.print(k,m,f,s);
#

function user.main():void;	# 0  (main:void)
#Illustrate the modules loaded after start 	# 1  (_2:str)
    k := inspect.getKind();	# 2 INSPECTgetkind (k:bat[:void,:str])
    m := inspect.getModule();	# 3 INSPECTgetAllModules (m:bat[:void,:str])
    f := inspect.getFunction();	# 4 INSPECTgetAllFunctions (f:bat[:void,:str])
    f1 := bat.reverse(f);	# 5 BKCreverse (f1:bat[:str,:void])<-(f:bat[:void,:str])
    f2 := algebra.ssort(f1);	# 6 ALGssort (f2:bat[:str,:void])<-(f1:bat[:str,:void])
    fs := bat.reverse(f2);	# 7 BKCreverse (fs:bat[:void,:str])<-(f2:bat[:str,:void])
    s := inspect.getSignatures();	# 8 INSPECTgetAllSignatures (s:bat[:void,:str])
    a := inspect.getAddresses();	# 9 INSPECTgetAllAddresses (a:bat[:void,:str])
    io.table(fs,k,m,s,a);	# 10 IOttable (_10:int)<-(fs:bat[:void,:str])(k:bat[:void,:str])(m:bat[:void,:str])(s:bat[:void,:str])(a:bat[:void,:str])
end main;	# 11  

# 00:03:55 >  
# 00:03:55 >  Done.
# 00:03:55 >  

